src/forms/Textarea.jsx

```diff
@@ -60,6 +64,10 @@ export class Textarea extends React.PureComponent {
 	 */
 	componentDidUpdate(prevProps) {
 		if (this.props.value !== prevProps.value) {
+			if (this.props.textarearef) {
+				autosize.update(this.props.textarearef);
+				return;
+			}
 			autosize.update(this.textarea);
```

Comment:
it doesn't look like `this.textarea` will be defined when `textarearef` is passed in as a prop - we should probably avoid calling `autosize.update` on `undefined`

Reply:
It shouldn't reach that code, there's a return in the block above 🙂